apps/ehr/tests/e2e/specs/in-person/medicalConditions.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { Page, test } from '@playwright/test';
+import { DateTime } from 'luxon';
+import { CssHeader } from 'tests/e2e/page/CssHeader';
+import { openInPersonProgressNotePage } from 'tests/e2e/page/in-person/InPersonProgressNotePage';
+import { MedicalConditionsPage } from 'tests/e2e/page/MedicalConditionsPage';
+import { SideMenu } from 'tests/e2e/page/SideMenu';
+import { ResourceHandler } from '../../../e2e-utils/resource-handler';
+
+const PROCESS_ID = `medicalConditions.spec.ts-${DateTime.now().toMillis()}`;
+const resourceHandler = new ResourceHandler(PROCESS_ID, 'in-person');
+const MEDICAL_CONDITION = 'Paratyphoid fever A';
```

Comment:
can we get this from ottehr spec somehow? if a hosted ottehr env doesn't have this value it will break the tests